{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@kernel.org",
      "ID": "CVE-2025-38570"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "In the Linux kernel, the following vulnerability has been resolved:\n\neth: fbnic: unlink NAPIs from queues on error to open\n\nCI hit a UaF in fbnic in the AF_XDP portion of the queues.py test.\nThe UaF is in the __sk_mark_napi_id_once() call in xsk_bind(),\nNAPI has been freed. Looks like the device failed to open earlier,\nand we lack clearing the NAPI pointer from the queue."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://git.kernel.org/stable/c/21d3f8441c7f317b93ba6a8029610c8b7e3773db",
          "refsource": "",
          "tags": [],
          "url": "https://git.kernel.org/stable/c/21d3f8441c7f317b93ba6a8029610c8b7e3773db"
        },
        {
          "name": "https://git.kernel.org/stable/c/4b31bcb025cb497da2b01f87173108ff32d350d2",
          "refsource": "",
          "tags": [],
          "url": "https://git.kernel.org/stable/c/4b31bcb025cb497da2b01f87173108ff32d350d2"
        },
        {
          "name": "https://git.kernel.org/stable/c/4b59f9deff3bdb52b223c85048f1d2924803b817",
          "refsource": "",
          "tags": [],
          "url": "https://git.kernel.org/stable/c/4b59f9deff3bdb52b223c85048f1d2924803b817"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2025-08-19T17:15Z",
  "publishedDate": "2025-08-19T17:15Z"
}